src/explore-education-statistics-admin/src/services/releaseAncillaryFileService.ts

```diff
@@ -51,7 +51,9 @@ const releaseAncillaryFileService = {
     data.append('file', request.file);
 
     const file = await client.post<AncillaryFileInfo>(
-      `/release/${releaseId}/ancillary?name=${request.name}`,
+      `/release/${releaseId}/ancillary?name=${encodeURIComponent(
```

Comment:
I don't think we need to actually do this directly, You can pass a third `options` param to `client.post`, which can contain some `params` (query parameters). Really we should be doing this in lots of other places as well.

Reply:
done